### PR TITLE
24107 Added designations for continued in business types

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#24107

*Description of changes:*
- app version = 5.5.10
- added designations for continued in business types (C, CBEN, CCC, CUL)
- imported latest shared enums

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-779-en1w3lo6.web.app

**Comment by severinbeauvais** [comment]
### Screenshots

Name Change - CCC
image

Name Change - ULC
image

Restore - C
image

Restore - CBEN
image

**Comment by severinbeauvais** [comment]
Do not merge yet. I will update https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts and import the latest enums package in Namerequest UI as part of this PR.

File: app/package.json
Comment (severinbeauvais) [inline app/package.json:20]: This is to pull in the latest NR request type codes (as added in bcgov/entity#23382).

These codes are not used in Namerequest UI but it's good to keep things up to date.

File: app/src/list-data/designations.ts
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:30]: C = same as CR

File: app/src/list-data/designations.ts
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:31]: CBEN = same as BC (Benefit Company)

File: app/src/list-data/designations.ts
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:47]: CCC = same as CC

File: app/src/list-data/designations.ts
Comment (severinbeauvais) [inline app/src/list-data/designations.ts:56]: CUL = same as ULC

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-779-en1w3lo6.web.app

**Comment by severinbeauvais** [comment]
This is ready for your review, and I'd love to get this merged and into Dev asap (to unblock further testing). Thanks!

**Review by eve-git — APPROVED** [review]
LGTM

**Pull request merged**
